package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "holiday-udp",
-  "version": "0.1.2",
+  "version": "2.0.0",
   "description": "Sends frames to the MooresCloud Holiday UDP API",
+  "type": "module",
```

Comment:
Saw the chatter on Mastodon, hope you don't mind drive-by feedback :)

Perhaps worth [adding `engines`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines)? I find with the Holiday I touch it so infrequently, there's basically no chance I remember which version of node I was working with last time ;)

Reply:
Everyone is more than welcome [with construtive feedback = that's one of the beuties of open-source]!

Great idea. [I've added this](https://github.com/garthk/holiday-udp/pull/2/commits/e30da7f3dd69830bc1728ecfbd1a32cac9da0c7d). Should I set it from what I'm using or what "should be" the minium? The only new requirement is `Buffer.from()` which [according to the docs](https://nodejs.org/api/buffer.html#static-method-bufferfromarray) was added to Node v5.10.0.

I guess as the _new_ lock file has `"lockfileVersion": 3` in it, then NPM >= 3 is also a requirement?

I've also tested with:
- Node v21.0.0
- NPM v10.2.0

Reply:
A range is kinder for consumers, couple of thoughts on the lower boundary though:

* I think the `type:module` behaviour flip in v12 might be significant to this change?
* [v18 is the oldest supported version](https://nodejs.org/en/about/previous-releases) so there's a case to use that as the minimum regardless

Since engines will only throw a warning anyway, I'd be inclined to bump the minimum to v18 and make a note about why.